docs/developer/getting-started/monorepo-packages.asciidoc

```diff
@@ -34,7 +34,7 @@ Non Bazel packages can be watched independently with
 
 [source,bash]
 ----
-yarn kbn watch -i PACKAGE_NAME
+yarn kbn run watch -i PACKAGE_NAME
```

Comment:
The actual command was `kbn:watch`, do we want to rename those scripts to `watch` in this PR too? Do they actually exist?

Reply:
Two left - one in kbn/pm and one in kibana's package.json.  The former hasn't been converted to use bazel.

@mistic do you have thoughts here?

Reply:
I think we can remove both  `kbn:watch` scripts and also the documentation about the command \cc @tylersmalley

Reply:
Pushed with f39f9fc3e7dbef2e9dc6aa6a5e0e6363fa6e050d.  Are we sure about the kbn:watch script in kbn/pm?  It looks like there's still a use for it, with it not being a part of bazel.

Reply:
I think its safe to remove this, I don't believe we want to support it from now on on the kbn watch step \cc @tylersmalley